mlir/include/mlir/Dialect/Bufferization/Transforms/Passes.h

```diff
@@ -46,7 +46,8 @@ std::unique_ptr<Pass> createLowerDeallocationsPass();
 /// Adds the conversion pattern of the `bufferization.dealloc` operation to the
 /// given pattern set for use in other transformation passes.
 void populateBufferizationDeallocLoweringPattern(
-    RewritePatternSet &patterns, func::FuncOp deallocLibraryFunc);
+    RewritePatternSet &patterns,
+    const llvm::DenseMap<Operation *, func::FuncOp> &deallocHelperFuncMap);
```

Comment:
Add comment for `deallocHelperFuncMap` and mention that this is mapping from symbol tables to dealloc functions.

Reply:
DONE